src/tiles/grid/item-tile.ts

```diff
@@ -19,91 +16,82 @@ export class ItemTile extends LitElement {
+    const itemTitle = this.model?.title;
     const itemCreator = this.model?.creator;
     return html`
       <div id="container">
-        <div id="title-image-container">
-          <h1 id="item-title" title=${itemTitle}>${itemTitle}</h1>
-          <div id="item-image-container">
-            <item-image .model=${this.model} .baseImageUrl=${this.baseImageUrl}>
-            </item-image>
-          </div>
-          <div class="item-creator">
-            <div class="truncated">
-              ${itemCreator
-                ? html`<span>by&nbsp;${itemCreator}</span>`
-                : nothing}
-            </div>
+        <div id="inner-wrapper">
+          <div class="title-wrapper">
+            <h1 id="item-title" title=${ifDefined(itemTitle)}>${itemTitle}</h1>
           </div>
-        </div>
 
-        <div class="hr"></div>
-
-        <div id="item-stats-container">
-          <div id="stats-holder">
-            <div class="col">
-              <mediatype-icon
-                .mediatype=${this.model?.mediatype}
-                .collection=${this.model?.collections}
-                style="--iconHeight:25px; --iconWidth:25px;"
-              >
-              </mediatype-icon>
-            </div>
-            <div class="col">
-              ${viewsIcon}
-              <p class="status-text">
-                ${formatCount(this.model?.viewCount, 'short', 'short')}
-              </p>
+          <div class="image-wrapper">
+            <div id="item-image-container">
+              <item-image .model=${this.model} .baseImageUrl=${
+      this.baseImageUrl
+    }>
+              </item-image>
             </div>
-            <div class="col">
-              ${favoriteFilledIcon}
-              <p class="status-text">
-                ${formatCount(this.model?.itemCount, 'short', 'short')}
-              </p>
-            </div>
-            <div class="col">
-              ${reviewsIcon}
-              <p class="status-text">
-                ${formatCount(this.model?.favCount, 'short', 'short')}
-              </p>
+            <div class="item-creator">
+              <div class="truncated">
+                ${
+                  itemCreator
+                    ? html`<span>by&nbsp;${itemCreator}</span>`
+                    : nothing
+                }
+              </div>
             </div>
           </div>
         </div>
+
+        <tile-stats 
+          .mediatype=${this.model?.mediatype}
+          .viewCount=${this.model?.viewCount}
+          .favCount=${this.model?.favCount}
+          .commentCount=${this.model?.commentCount}>
+        </tile-stats>
+        </div>
       </div>
     `;
   }
 
   static get styles(): CSSResultGroup {
-    const cornerRadiusCss = css`var(--collectionTileCornerRadius, 4px)`;
-
     return css`
       #container {
         background-color: #ffffff;
-        border-radius: ${cornerRadiusCss};
+        border-radius: var(--collectionTileCornerRadius, 4px);
```

Comment:
as I remember, we don't want to use px unit in CSS instead we use rem units.

Reply:
it was stated in the mockups though to use `px`

Reply:
@nsharma123 yep, for pixel perfect design parts like border-radius, we can use px. for text, rule of thumb is using rem.